ietf/templates/doc/document_html.html

```diff
@@ -131,8 +131,8 @@
                             <i class="bi bi-arrow-bar-up"></i>
                     </button>
                     <div class="pt-2 pt-lg-3 px-md-2 px-lg-3">
-                        <p>
-                            <a href="{% url 'ietf.doc.views_doc.document_main' name=doc.canonical_name %}">
+                        <a class="btn btn-primary btn-sm" href="{% url 'ietf.doc.views_doc.document_main' name=doc.canonical_name %}">Datatracker</a>
```

Comment:
Should this be changed to point to the same version of the metadata page that you are looking at on the /html/ page? Or are you leaving it such that it always points to the current version for a reason?

Reply:
I have no preference.